webui/react/src/utils/data.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
ooh this might make it easy for us to ignore `any` usage. feel free to ignore this though.